CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Changelog for the HuBMAP CCF ASCT+B Reporter
 
+## 2.2.0 - Pre-release
```

Comment:
I like it, great idea! We'll do this at the start of the 2.3.0 sprint next time :-)